src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Execute.cs

```diff
@@ -152,6 +166,9 @@ private int ExecuteVisualStudio2013(ref Guid pguidCmdGroup, uint commandId, uint
             return result;
         }
 
+        private static readonly Guid _asyncFindRefsCommandGroup = new Guid("cc5d21e2-c5cf-49d0-bf20-24bb7eb776d8");
```

Comment:
David is defining it for now in his VSIX.  When this becomes part of the editor API it will be in an appropriate group.